eng/source-build-release-notes.md

```diff
@@ -0,0 +1,22 @@
+You can build $RELEASE_NAME from the repository by cloning the release tag `$TAG` and following the build instructions in the [main README.md](https://github.com/dotnet/dotnet/blob/$TAG/README.md#building).
+
+```sh
+git clone --branch $TAG --depth 1 https://github.com/dotnet/dotnet
+(cd dotnet && ./build.sh)
```

Comment:
```suggestion
(cd dotnet && ./prep.sh && ./build.sh)
```

Reply:
Is this so? I know we've just discussed that `prep.sh` is not run by partners? @MichaelSimons?

Reply:
@MichaelSimons can you weigh in please? I am not sure if the `prep.sh` should be part of the instructions on dotnet/dotnet releases

Reply:
I don't think we should be including build instructions in the announcement.  There are too many scenarios to do justice here.  We should be linking to documentation within the VMR source.  

Distro maintainers are likely going to be supplying a previously source-build SDK and artifacts.  Others will need to run prep.